src/formats/jpeg/frame.zig

```diff
@@ -0,0 +1,247 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+
+const buffered_stream_source = @import("../../buffered_stream_source.zig");
+const Image = @import("../../Image.zig");
+const ImageReadError = Image.ReadError;
+
+const Markers = @import("utils.zig").Markers;
+const FrameHeader = @import("frame_header.zig");
+const QuantizationTable = @import("quantization.zig").Table;
+const HuffmanTable = @import("huffman.zig").Table;
+const color = @import("../../color.zig");
+
+const IDCTMultipliers = @import("utils.zig").IDCTMultipliers;
+const MAX_COMPONENTS = @import("utils.zig").MAX_COMPONENTS;
+const MAX_BLOCKS = @import("utils.zig").MAX_BLOCKS;
+const MCU = @import("utils.zig").MCU;
+
+const Self = @This();
+allocator: Allocator,
+frame_header: FrameHeader,
+quantization_tables: *[4]?QuantizationTable,
+dc_huffman_tables: [2]?HuffmanTable,
+ac_huffman_tables: [2]?HuffmanTable,
+
+const JPEG_DEBUG = false;
```

Comment:
I am wondering if would be better to only set `JPEG_DEBUG` once in `jpeg.zig` and the other file just import that symbol so it only need to be modified in one place to get debug information when working on the JPEG format.

Reply:
I remember that when i was refactoring there was a "JPEG_VERY_DEBUG" variable. 
The downside is that you will have too many things printed on the terminal. I think with the current approach you can decide what to print on a granular way, without commenting prints I'd say.